src/shared/options/readOptions.test.ts

```diff
@@ -0,0 +1,86 @@
+import { describe, expect, it, vi } from "vitest";
+
+import { readOptions } from "./readOptions.js";
+
+const emptyOptions = {
+	author: undefined,
+	base: undefined,
+	createRepository: undefined,
+	description: undefined,
+	email: undefined,
+	excludeCompliance: undefined,
+	excludeContributors: undefined,
+	excludeLintJson: undefined,
+	excludeLintKnip: undefined,
+	excludeLintMd: undefined,
+	excludeLintPackageJson: undefined,
+	excludeLintPackages: undefined,
+	excludeLintPerfectionist: undefined,
+	excludeLintSpelling: undefined,
+	excludeLintYml: undefined,
+	excludeReleases: undefined,
+	excludeRenovate: undefined,
+	excludeTests: undefined,
+	funding: undefined,
+	owner: undefined,
+	repository: undefined,
+	skipGitHubApi: false,
+	skipInstall: false,
+	skipRemoval: false,
+	skipRestore: undefined,
+	skipUninstall: false,
+	title: undefined,
+};
+
+const mockOptions = {
+	base: "prompt",
+	github: "mock.git",
+	repository: "mock.repository",
+};
+
+vi.mock("./getPrefillOrPromptedOption.js", () => ({
+	get getPrefillOrPromptedOption() {
+		return vi.fn(() => "mock");
+	},
+}));
```

Comment:
[Cleanup] There's no need to `get` or `vi.fn` if the mocks aren't being used as mocks anywhere. I've just used that strategy elsewhere because I don't know of a better/cleaner way to provide function mocks to mocked modules (😩).

```suggestion
vi.mock("./getPrefillOrPromptedOption.js", () => ({
	getPrefillOrPromptedOption: () => "mock",
}));
```

(here and elsewhere)